### PR TITLE
Update and corrections for french translation

### PR DESCRIPTION
**Pull request by ghost**
Update and corrections for french translation
Some strings (69, 803, 831) looked a bit odd, they were splitted in multiples quotes.
Please  check if I did right rewritting them

### CODE REVIEW

File: locale/fr.po [hunk 1]
```diff
@@ -547,7 +546,7 @@ msgstr "Mine ton premier diamant."
 #: init.lua
 #, fuzzy
 msgid "Girl's Best Friend"
-msgstr ""
+msgstr "Girl's Best Friend"
```
Comment (kaeza) [inline locale/fr.po:542]: Is this correct?

**Comment by kaeza** [comment]
Thank you for your time.

>Some strings (69, 803, 831) looked a bit odd, they were splitted in multiples quotes.

Standard gettext tools break lines at 80 columns (and also when there's a newline character in the string IIRC). Both forms are equivalent but the multi-line form is because the standard tools use it. It's just a matter of preference though, and I'll keep this as it is until somebody complains.

It seems there are some strings which are unchanged from the source strings. Can you review these? Things like "Place 100 torches".

Can you also review all strings marked as "fuzzy" and unmark them if they are correct?

Thank you again.

**Comment by ghost** [comment]
I rechecked every strings marked as "fuzzy" most of them seemed ok, so I unmarked them
"Place 100 torches" is actually the same in french as well as every strings unchanged from the source strings as far as I could see, exept maybe for "Girl's Best Friend" wich I think should be kept in english too since it is a musical reference (or at least I think it is) and french translation would just sound like a sexist statement so... I would prefer not to

I have a doubt about "Place 20 coal checkers." and "Place 20 iron checkers." because I don't use moreblock so I'm not sure what checkers actually are (I assumed it was "pions" as in "pieces of a game").

**Comment by kaeza** [comment]
Thanks for reviewing.

>I rechecked every strings marked as "fuzzy" most of them seemed ok, so I unmarked them
"Place 100 torches" is actually the same in french as well as every strings unchanged from the source strings as far as I could see, exept maybe for "Girl's Best Friend" wich I think should be kept in english too since it is a musical reference (or at least I think it is) and french translation would just sound like a sexist statement so... I would prefer not to

Ah okay. That's fine with me.

>I have a doubt about "Place 20 coal checkers." and "Place 20 iron checkers." because I don't use moreblock so I'm not sure what checkers actually are (I assumed it was "pions" as in "pieces of a game").

They are tiles with a checkerboard pattern. Here's the coal and iron checkers textures for reference, taken directly from moreblocks:

 

Does the string make sense or does it need change?

If it's ok with you I can rebase and make a pull request to the original repo, since the branch this is based off was merged prematurely.

**Comment by ghost** [comment]
Thanks I looked at the translation from moreblock and I was wrong. It is now corrected.

> If it's ok with you I can rebase and make a pull request to the original repo, since the branch this is based off was merged prematurely.

Yes of course it's ok

**Comment by kaeza** [comment]
Thank you again!

The PR is at minetest-mods/awards#41.

**Pull request closed without merging**